packages/documentation/src/stories/components/dialog/dialog.stories.ts

```diff
@@ -144,11 +190,13 @@ const Template = {
         data-position="${args.position}"
         data-animation="${args.animation}"
         open="${args.open || nothing}"
+        aria-labelledby="dialog-title"
+        aria-describedby="dialog-description"
       >
         <form method="dialog" class="dialog-grid">
           ${postDialogIcon}
-          <h3 class="dialog-header">${args.title}</h3>
-          <div class="dialog-body"><p>${args.content}</p></div>
+          <h3 class="dialog-header" id="dialog-title">${args.title}</h3>
+          <div class="dialog-body"><p id="dialog-description">${args.content}</p></div>
```

Comment:
Can you explain, why the IDs are essential?
Imo, this makes the use of the component unnecessarily complicated 😞